qa/rpc-tests/bip100-sizelimit.py

```diff
@@ -10,11 +10,18 @@
 from test_framework.util import *
 
 from decimal import Decimal
-
 CACHE_DIR = "cache_bigblock"
+UAHF_INITIAL_MAX_BLOCK_SIZE = 8000000
```

Comment:
Could be replaced with `from test_framework.mininode import MAX_BLOCK_SIZE`

Reply:
Done